src/velero/tasks/deployment-validation.yaml

```diff
@@ -0,0 +1,79 @@
+variables:
+  - name: BACKUP_NAME
+    default: backup-test
+tasks:
```

Comment:
Any reason to not inline these in `src/velero/tasks.yaml`? Having a new `tasks` folder isn't wrong, but seems unnecessary here since the top level tasks is just doing an include?

Reply:
The intention was to keep the test case output (health check vs backup functionality) separate while still matching the top level `validate` task name. I think if I in-line them both under the `validation` task, they would both look like part of the same test when they really do very different things. 

I'm certainly open to changing it if that's the pattern we're following elsewhere though.

Reply:
Ah gotcha. So you should still be able to achieve the same goal, but all in one file for `tasks.yaml`. Rough example:
```yaml
variables:
  - name: BACKUP_NAME
    default: backup-test

tasks:
  - name: validate
    actions:
      - task: health-check
      - task: execute-backup
  - name: health-check
    actions:
      - description: Velero StatefulSet Health Check
        wait:
          cluster:
            kind: Deployment
            name: velero
            namespace: velero
  - name: execute-backup
    actions:
...
```

Basically just adding them as separate tasks, but in the same file still so that we don't need the include layer?

Reply:
Doh!
Somehow I didn't realize you could reference task defs from the same file rather than an import.

Agreed that's cleaner -- I'll update accordingly